src/routes/volume/VolumeActions.js

```diff
@@ -181,7 +177,7 @@ function actions({ selected, engineImages, showAttachHost, detach, showEngineUpg
     }
   })
 
-  availableActions.push({ key: 'expandVolume', name: 'Expand Volume', disabled: !(selected.actions && (Object.keys(selected.actions).includes('attach'))) || (selected.conditions && selected.conditions.scheduled && selected.conditions.scheduled && selected.conditions.scheduled.status.toLowerCase() === 'false') || isRWXVolumeAndAttached() })
+  availableActions.push({ key: 'expandVolume', name: 'Expand Volume', disabled: selected.state !== 'detached' || (selected.conditions && selected.conditions.scheduled && selected.conditions.scheduled && selected.conditions.scheduled.status.toLowerCase() === 'false') })
```

Comment:
```suggestion
  availableActions.push({ key: 'expandVolume', name: 'Expand Volume', disabled: selected.state !== 'detached' || (selected.conditions && selected.conditions.scheduled && selected.conditions.scheduled.status.toLowerCase() === 'false') })
```

Reply:
Thanks for your reminding, mistakes have been corrected.